index.html

```diff
@@ -48,35 +48,39 @@
 
           <div class="subscribe-form">
             <h2>Coming to Seattle and Berlin in 2016</h2>
-            <p class="stay-tuned">Stay tuned for more info!</p>
 
-<!--
-             <form>
+             <form id="newsletter-signup" name="newsletter-signup">
               <fieldset>
                 <legend><span>Subscribe for updates</span></legend>
                 <ol>
-                  <li>
-                    <label for="subscribe-email">E-mail</label>
-                    <input type="email" id="subscribe-email" name="email" required placeholder="you@example.com">
+                  <li class="error"></li>
+                  <li id="li_email">
+                    <label for="id_email">E-mail</label>
+                    <input type="email" id="newsletter-email" name="email" required placeholder="you@example.com">
                   </li>
-                  <li class="form-fine-print">
+                  <li id="li_newsletters">
+                    <label for="newsletters">I want to hear about...</label>
+                    <input type="radio" name="newsletters" value="view-source-conference-north-america" required>Seattle</input>
```

Comment:
`<input>` should be a self-closing tag (`</input>` on the end is invalid). Both `Seattle` and `Berlin` should probably be labels here too, and `I want to hear about...` should be a heading?